docs/reference/analysis/tokenfilters/synonym-tokenfilter.asciidoc

```diff
@@ -21,7 +21,8 @@ PUT /test_index
                 "filter" : {
                     "synonym" : {
                         "type" : "synonym",
-                        "synonyms_path" : "analysis/synonym.txt"
+                        "synonyms_path" : "analysis/synonym.txt",
+                        "tokenizer" : "whitespace"
```

Comment:
Could you please add a callout <1> to the example that you added?
 
Example is https://raw.githubusercontent.com/elastic/elasticsearch/master/docs/reference/analysis/analyzers/configuring.asciidoc